packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -42,12 +53,29 @@ contract PseZkVerifier is EssentialContract, IVerifier {
         // Do not run proof verification to contest an existing proof
         if (isContesting) return;
 
-        bytes32 instance = getInstance(prover, evidence);
+        ProofData memory data = abi.decode(evidence.proof, (ProofData));
+        bytes32 instance = getInstance(prover, data.txListHash, evidence);
+
+        // Verify blob
+
+        // Question: What if x is larger than the blob data size
+
+        // Question: how to calculate pointCommitment and pointProof
+        // offchain?
+        Lib4844.point_evaluation_precompile({
+            versionHash: blobVersionHash,
+            x: uint256(instance) % Lib4844.FIELD_ELEMENTS_PERBLOB,
```

Comment:
```suggestion
            `x: keccak(blobVersionHash, 0)` % Lib4844.BLS_MODULUS,
```

The `0` is not correct but would need some extra data exposed by the proof (or already in the proof, but would have to be extracted, not sure what makes the most sense).